pkg/cmd/config_windows.go

```diff
@@ -42,3 +44,32 @@ func (c *Config) readPassword(prompt string) (password string, err error) {
 	password = string(passwordBytes)
 	return
 }
+
+func (c *Config) windowsVersion() (map[string]interface{}, error) {
+	registryKey, err := registry.OpenKey(registry.LOCAL_MACHINE, `SOFTWARE\Microsoft\Windows NT\CurrentVersion`, registry.QUERY_VALUE)
+	if err != nil {
+		return nil, err
+	}
+	windowsVersion := make(map[string]interface{})
+	for _, name := range []string{
+		"CurrentBuild",
+		"CurrentVersion",
+		"DisplayVersion",
+		"ProductName",
```

Comment:
Could we also include `"EditionID"`?

Use case(s):

- Some editions have more restrictions than others on how software can be installed.
- Enterprise and Server editions have [built-in methods to block telemetry](https://docs.microsoft.com/en-us/windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services) but other editions have to use third-party scripts etc.

Reply:
Of course! Thanks for the review :)